mice-and-cats-java/src/main/java/foop/views/StartView.java

```diff
@@ -0,0 +1,97 @@
+package foop.views;
+
+import foop.message.AvailableGamesMessage;
+import foop.message.InitialMessage;
+import foop.message.JoinGameMessage;
+import lombok.Getter;
+import lombok.extern.slf4j.Slf4j;
+
+import javax.swing.*;
+import java.awt.*;
+
+@Slf4j
+public class StartView extends JPanel implements View {
+
+
+    private final GameFrame frame;
+    @Getter
+    private JList<AvailableGamesMessage.Game> lobbyList;
+
+    public StartView(GameFrame frame) {
+        this.frame = frame;
+        render();
+    }
+
+
+    public void render() {
```

Comment:
This doesn't need to be public and if you inline it into the constructor you can make `lobbyList` `final`